common/rootfs.yaml

```diff
@@ -62,7 +63,7 @@ actions:
       set -e
 
       rm -f /etc/resolv.conf
-      rsetup update_hostname radxa
+      rsetup update_hostname {{ $board }}
```

Comment:
Change host name at the end of [image.yaml](https://github.com/radxa-repo/rbuild/blob/087be144910ec3ff9b50156ff774d931f67124e2/common/image.yaml#L144). Changes here will be saved in the cached rootfs.